src/extra/recipes/cookiebox.py

```diff
@@ -1095,6 +1186,7 @@ def apply_correction(tof_id, tof_trace):
             #o = o - self.offset[tof_id][None, None, :]
             # apply Jacobian
             o = o*self.jacobian[tof_id][None, None, :]
+            o[np.isnan(o)] = 0.0
```

Comment:
Maybe it would be safer to use `~np.isfinite(o)`? I've used `isnan` in the past and inevitably been tripped up by infs. Could also use `np.nan_to_num(o, copy=False)`: https://numpy.org/doc/2.0/reference/generated/numpy.nan_to_num.html